source/shared/index/_intro.slim

```diff
@@ -2,7 +2,7 @@
 	a name="intro-en"
 
 	p
-		a href="#intro-de" onclick="javascript:$('#content-en').hide();$('#content-de').show();eventBox.resizeTriangle();" Zur Deutschen Version
+		a class="change-language" href="#intro-de" onclick="javascript:$('#content-en').hide();$('#content-de').show();eventBox.resizeTriangle();" Zur Deutschen Version
```

Comment:
CSS classes should describe something about the **visual** properties of an element not the functional properties.  can you think of a better name?